miniapps/meshing/pmesh-optimizer.cpp

```diff
@@ -416,6 +419,12 @@ int main (int argc, char *argv[])
    x.SetTrueVector();
    x.SetFromTrueVector();
 
+   // Optional. Uncomment below to use a Kershaw transformed mesh. Input mesh
+   // be a Cartesian-aligned mesh where nx is mutiple of 6, and ny and nz are
+   // multiples of 2.
+   // common::KershawTransformation kershawT(pmesh->Dimension(), 0.3, 0.3);
```

Comment:
Instead of introducing commented code, can this be enabled via a command line argument?

Reply:
Yeah.. I had that earlier but decided against it just because this miniapp has exploded already.. we mainly want to have the sample run because it was used for some results in a paper. 
Maybe I can get rid of these comments completely and just change the sample run to indicate that the user should use a Kershaw transformed mesh (possibly generated using the updates to `transformation` in mesh-explorer).